package.json

```diff
@@ -5,6 +5,11 @@
   "scripts": {
     "docusaurus": "docusaurus",
     "start": "docusaurus start",
+    "start-bg": "nohup yarn start --port 3000 > docs.log 2>&1 & echo $! > run.pid",
+    "dc-process": "echo Docusaurus running on process: $(cat run.pid)",
+    "dc-start": "yarn start-bg && yarn dc-process",
+    "dc-stop": "kill -9 $(lsof -t -i:3000)",
+    "dc-restart": "yarn dc-stop && yarn dc-start",
```

Comment:
to be honest, i'm a bit hesitant to add more scripts here. it's probably already a little too crowded and could use some cleaning up lol.

is there a way we can put these into the `devcontainer.json` file, where they might be a little more "at home"?

Reply:
Yea I can move them to shell scripts, package.json is crowded already